.vscode/settings.json

```diff
@@ -0,0 +1,7 @@
+{
+    "rubyLsp.indexing": {
+        "includedPatterns": [
+            "test/dummy/**/*.rb"
+        ]
+    }
```

Comment:
We need to ensure that the dummy files are indexed to be able to test our own features. A recent optimization in the LSP made us fully ignore the entire test directory straight away.